Gruntfile.js

```diff
@@ -31,6 +31,9 @@ module.exports = function(grunt) {
           'lib/iced-coffee-script.js':
              'iced-coffee-script/extras/iced-coffee-script-1.8.0-c.js',
           'lib/jquery.js' : 'jquery/dist/jquery.js',
+          'lib/jquery-ui.min.js' : 'jquery-ui/jquery-ui.min.js', 
```

Comment:
We don't need .min.js here - we minify on our own.  Link the unminified version.

Also, remove the minified version below.